src/main/java/frc/robot/RobotContainer.java

```diff
@@ -144,10 +150,11 @@ public Command getAutonomousCommand() {
 
   private void configureBindings() {
     intake.whileTrue(new IntakeCommand());
-    climbExtension.whileTrue(new ClimbExtensionCommand());
-    climbRetraction.whileTrue(new ClimbRetractionCommand());
     yButton.whileTrue(new InstantCommand(() -> s_Swerve.zeroHeading()));
     s_DefaultTeleopSub.setDefaultCommand(new DefaultTeleop(driver, operator));
+    upDPad.whileTrue(new InstantCommand(() -> climbingSubsystem.climbControl(0.5, 0.5)));
+    downDPad.whileTrue(new InstantCommand(() -> climbingSubsystem.climbControl(-0.5, -0.5)));
```

Comment:
Let's comment these dpad mappings out for now. If we decide there's a use case for them, we'll have to change them a bit anyway.